src/features/auth/pages/login/schema/loginSchemaa.test.ts

```diff
@@ -0,0 +1,50 @@
+import { ValidationError } from 'yup'
+
+import { loginSchema } from 'features/auth/pages/login/schema/loginSchemaa'
+
+describe('loginSchema', () => {
+  describe('should validate', () => {
```

Comment:
Je pense que les describe ne sont pas utiles, surtout le premier qui a qu'un seul teste. 
On peut juste mettre les it avec "it should validate ..." et "it should invalidate..." ?